iFixit/react-commerce
# Add TikTok to footer

### Overview:

![socials](https://user-images.githubusercontent.com/95656772/198393177-bedc05fd-1c13-4692-90da-05cae20a9f8f.png)

---

### Changes Made:
1. Added a TikTok SVG from [FontAwesome](https://fontawesome.com/icons/tiktok)
2. Added TikTok-related fields to **SocialMediaAccounts**

---
>### CR/QA Notes:
>
> **CR**: Review to https://github.com/iFixit/react-commerce/pull/905/commits/f03f53c828f1f6afd9ee5e12b4b52d8c3f917193
>
>**QA**:
>
>1. Navigate to [the Vercel preview](https://react-commerce-prod-d2x5n7xld-ifixit.vercel.app/)
>2. Check that **TikTok** is in the footer
>3. Ensure it links to our social media page
>
---

cr_req 2
Connects #44933
